versioned_staging/Quickstart.mdx

```diff
@@ -28,7 +28,7 @@ Do the same for winget. Type `winget` in the search bar, and click on the `Get`
 
 Open PowerShell from the Start menu and type `winget install Enarx` to install Enarx.
 
-![windows install enarx](https://enarx.dev/img/install/win_install_enarx.png?raw=true)
+![windows install enarx](/img/install/win_install_enarx.png)
```

Comment:
Thanks for adding the relative paths now. I added full URLs before because it was breaking yarn (missing images...)